modules/mtl-proposal-form/mtl-proposal-form.php

```diff
@@ -265,6 +265,11 @@ function mtl_proposal_form_output( $atts ){
 			}
 			$output .= '}; </script>'."\r\n";
 			$output .= '<script type="text/javascript"> '.$output_later.' var centerLon = "'.$mtl_options['mtl-center-lon'].'"; var centerLat = "'.$mtl_options['mtl-center-lat'].'"; var standardZoom = '.$mtl_options['mtl-standard-zoom'].'; </script>'."\r\n";
+
+			// TODO make this a setting
+			$output .= '<script type="text/javascript"> var countrySource = \''.str_replace(array("\r", "\n"), "", file_get_contents('http://localhost/wordpress/wp-content/uploads/VG5000_LAN.geojson')).'\';'."\r\n";

```

Comment:
Why are there static links here? If the GeoJSON boundary files are to be used in static form within the theme, they should be included to the theme directly, which is completely fine as a temporary solution. But IMHO there should be no static links to anywhere above theme folder level.